aten/src/ATen/test/cuda_bfloat16_test.cu

```diff
@@ -0,0 +1,98 @@
+#include <gtest/gtest.h>
+
+#include <ATen/ATen.h>
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/NumericLimits.cuh>
+
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
+#include <cuda.h>
+#include <cuda_bf16.h>
+#include <cuda_runtime.h>
+
+#include <assert.h>
+
+using namespace at;
+
+__device__ void test(){
```

Comment:
should this be a templated test rather than copy-paste? You can have BFloat16, __nb_bfloat16, __float2bfloat16 and their half counterparts as template parameters, is there anything else?

Reply:
I agree. Fixed.